en/advanced_config/pid_tuning_guide_multicopter.md

```diff
@@ -1,217 +1,198 @@
 # Multicopter PID Tuning Guide
 
-This tutorial explains how to tune the PID loops on PX4 for all [multi rotor setups](../airframes/airframe_reference.md#copter) (Quads, Hexa, Octo etc).
+This tutorial explains how to tune the PID loops on PX4 for all [multicopter setups](../airframes/airframe_reference.md#copter) (Quads, Hexa, Octo etc).
 
-Tuning is required when creating a new airframe type or significantly modifying an existing supported frame. Generally if you're using a [supported configuration](../airframes/airframe_reference.md#copter) (e.g. using an airframe in [QGroundControl > Airframe](../config/airframe.md)) the default tuning should be acceptable (particularly for larger frames). 
+Tuning is required when creating a new airframe type or modifying an existing supported frame. Generally if you're using a [supported specific configuration](../airframes/airframe_reference.md#copter) (e.g. using an airframe in [QGroundControl > Airframe](../config/airframe.md)), where all components are fixed, the default tuning should work.
+To get the best performance it is usually good to tune a new vehicle, as for example using other ESCs or motors will require different tuning gains.

```

Comment:
@bkueng The paragraph here is a little confusing because 
- We don't define what might constitute "modifying an existing supported frame"
- "new vehicle" here is confusing because our airframe configurations do not specify a particular ESC or motor.  

I think we can address this with a slight refocus .... thoughts?

> Generally if you're using a [supported specific configuration](../airframes/airframe_reference.md#copter) (e.g. using an airframe in [QGroundControl > Airframe](../config/airframe.md)) the default tuning should be sufficient to fly the vehicle safely.
> To get the very best performance it is usually good to tune a new vehicle. For example, different ESCs or motors  require different tuning gains for optimal flight.

Reply:
I updated as suggested in https://github.com/PX4/px4_user_guide/pull/272/commits/7cabfa6cd1ea91f1b582c671f2c3dde6bb9fbefb